src/components/Profile/History/index.tsx

```diff
@@ -114,10 +115,12 @@ export default function HistoryPage({
     setRefetchJobs
   )
 
-  let defaultTabIndex = 0
-  defaultTab === 'ComputeJobs' ? (defaultTabIndex = 4) : (defaultTabIndex = 0)
```

Comment:
suggestion: would be nice to retain this functionality (or re-enable it)
We may want to link to e.g., the compute history or the download tab and could achieve this by simply passing the tab index here for example

`.../profile?defaultTab=2` could link to compute history
```tsx
useEffect(() => {
  setTabIndex(defaultTab)
}, [defaultTab])
```
something like this

Reply:
Implemented in https://github.com/deltaDAO/mvg-portal/pull/416